pkg/clusteragent/admission/mutate/tags_test.go

```diff
@@ -95,37 +95,6 @@ func Test_injectTagsFromLabels(t *testing.T) {
 	}
 }
 
-func Test_shouldInjectTags(t *testing.T) {
```

Comment:
Can we recreate this test to ensure that we're not regressing in the ability to set tags on pods?

Reply:
We are extending cases when tags are injected - before they were only injected if the label was set. That doesn't make sense. So now tags will be injected if: label is set, mutateUnlabelled is set and APM instrumentation is set. All these cases are cover by the unit test.